plugins/catalog-backend/src/database/metrics.ts

```diff
@@ -30,23 +29,22 @@ export function initDatabaseMetrics(knex: Knex) {
       help: 'Total amount of entities in the catalog. DEPRECATED: Please use opentelemetry metrics instead.',
       labelNames: ['kind'],
       async collect() {
-        const result = await knex<DbRefreshStateRow>('refresh_state').select(
-          'entity_ref',
-        );
-        const results = result
-          .map(row => row.entity_ref.split(':')[0])
-          .reduce((acc, e) => acc.set(e, (acc.get(e) || 0) + 1), new Map());
+        const results = await knex<DbSearchRow>('search')
+          .where('key', '=', 'kind')
+          .whereNotNull('original_value')
```

Comment:
I'd expect it to be best to use the case normalized value rather than the original one?

Reply:
We are? where's that?

Feels like it would be safer to use `value` either way?

Reply:
Just by virtue of the schemas using strict enums and we use ajv to match against those schemas.

https://github.com/backstage/backstage/blob/5c703d973459caebff85326bad875f5354d828ab/packages/catalog-model/src/schema/kinds/API.v1alpha1.schema.json#L40

Also I guess it's a matter of how dearly we want these metric values to actually be correct instead of forcibly lowercased.